s3-scan-tar/scanner.py

```diff
@@ -36,28 +38,28 @@ def __init__(self, filehandle, maxsize=None):
     def write(self, s) -> int:
         raise NotImplementedError("write() not supported")
 
-    def read(self, read_length) -> bytes:
+    def read(self, read_length: Any) -> bytes:
```

Comment:
```suggestion
    def read(self, read_length: int) -> bytes:
```

Eller er det noe jeg misforstår?